frontend/src/pages/Policy/policySections/ReformOpportunitiesTab.tsx

```diff
@@ -1,66 +1,63 @@
 import { Helmet } from 'react-helmet-async'
 import ResourceItem from '../policyComponents/ResourceItem'
-import {
-  effortsAndInitiatives,
-  legislativeActions,
-} from '../policyContent/ReformOpportunitiesContent'
+import { effortsAndInitiatives, legislativeActions } from '../policyContent/ReformOpportunitiesContent'
 import { OpenInNew } from '@mui/icons-material'
 import HetTextArrowLink from '../../../styles/HetComponents/HetTextArrowLink'
 
 export default function ReformOpportunitiesTab() {
-  return (
-    <>
-      <Helmet>
-        <title>Reform Opportunities - Health Equity Tracker</title>
-      </Helmet>
-      <h2 className='sr-only'>Reform Opportunities</h2>
-      <p>
-        Our data points to several reform opportunities, particularly in
-        enhancing community-based interventions, improving data collection on
-        race and ethnicity for non-fatal injuries, and addressing the root
-        causes of violence through equity-focused policies.
-      </p>
-      <section id='#city-and-county-level-reform-opportunities'>
-        <h3 className='my-0 text-title font-medium text-altGreen'>
-          Reform Opportunities at the County and City Levels
-        </h3>
-        <p>
-          <ul className='list-none'>
-            {effortsAndInitiatives.map((effortsAndInitiative, index) => (
-              <ResourceItem
-                key={index}
-                title={effortsAndInitiative.title}
-                description={effortsAndInitiative.description}
-                link={effortsAndInitiative.link}
+	return (
+		<>
+			<Helmet>
+				<title>Reform Opportunities - Health Equity Tracker</title>
+			</Helmet>
+			<h2 className='sr-only'>Reform Opportunities</h2>
+			<p>
```

Comment:
I missed this before, but we should remove the <ul> from the` <p>` tag. We can just apply the my-4 class directly to the `<ul>` to keep the same spacing.